test/one/profiler/test/Assert.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.profiler.test;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class Assert {
+
+    public static void contains(Output out, String regex) throws AssertionError {
+        if (!out.contains(regex)) {
+            throw new AssertionError("Expected: " + regex + "\nReceived out may be available in build/test/logs.");
+        }
+    }
+
+    public static void notContains(Output out, String regex) throws AssertionError {
+        if (out.contains(regex)) {
+            throw new AssertionError("Expected not: " + regex + "\nReceived out may be available in build/test/logs.");
+        }
+    }
+
+    public static void ratioGreater(Output out, String regex, double threshold) {
```

Comment:
I'd replace these methods with more generic `Assert.isGreater(double value, double threshold)`, so it could be reused in other cases without `Output`.
Calls to `Assert.ratioGreater(out, regex, threshold)` will then be replaced with `Assert.isGreater(out.ratio(regex), threshold)`

Reply:
Yes makes sense. This is from PR-812 again. Addressed and will reflect in next commit.